app/controllers/citizens/savings_and_investments_controller.rb

```diff
@@ -0,0 +1,58 @@
+module Citizens
+  class SavingsAndInvestmentsController < BaseController
+    before_action :bank_accounts, :back_link
```

Comment:
This isn't doing anything, as the method are lazy loading when first called.

Reply:
But they are not called anywhere

Reply:
Although I should use
```
    def back_link
      @back_link = citizens_own_home_path
```
instead of 
```
    def back_link
      @back_link ||= citizens_own_home_path
```
because these methods only serve to populate the @ variables so they can be used in the page

Reply:
OK - my bad .... though an alternative solution would be to replace `before_action` with `helper_method`.
```ruby
helper_method :bank_accounts, :back_link
```
That would make `bank_accounts` and `back_link` available to call from the views.

Reply:
👍 I'll use `helper_method`